.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # pin@v3.5.0
+        with:
+          fetch-depth: 0
+      - name: Create Release Artifacts
+        run: make release
+        env:
+          IMAGE_VERSION: ${{ github.ref_name }}
+      - name: Upload Release Artifacts
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            ./release/helm-chart-${{ github.ref_name }}.tgz
+            ./release/linode-blockstorage-csi-driver-${{ github.ref_name }}.yaml
+      - name: Docker Meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            linode/linode-blockstorage-csi-driver
+          tags: |
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+            type=semver,pattern={{raw}},value=${{ github.ref_name }}
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and Push to Docker Hub
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: "./app/linode/Dockerfile"
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
This action is not triggered on PRs, why do we want to filter this out?

Reply:
copy/paste error from the first attempt at this where i kept the pr/build process in the same file... great catch and ill drop it